package.json

```diff
@@ -10,6 +10,7 @@
   },
   "scripts": {
     "build": "scripts/build.sh",
+    "ci": "npm run lint && npm test && npm run coverage",
```

Comment:
yes i agree, i'm not against xunit but was against the previous implementation of it.